content/jury-advice.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Advice, Hints, and General Information from the NWERC Jury"
+weight: 6
+draft: false
+menu:
+  main:
+    name: Jury Advice
+parent: rules
+type: page
+---
+
+- Your submissions will be run multiple times,
+  on different input files.
+  The order in which the input files are being used is fixed
+  and does not change from one submission to another.
+
+  If your submission is incorrect,
+  the error message you get will be the error that occurred
+  on the first input file on which you failed.
+  E.g., if your submission is prone to crash but also incorrect,
+  your submission may be judged as either "Wrong Answer" or "Run Time Error",
+  depending on which is discovered first.
+
+- Your submission will have exactly one processor core
+  fully at its disposal while running.
+  You should not attempt to use multithreading in your submissions.
+  Attempts to start extra processes or threads may
+  lead to a judgement of "Run Time Error".
+
+- Each problem will have a stated time limit.
+  This time limit is per input file and refers to CPU time,
+  which includes time spent on slow parsing
+  (e.g., using `Scanner` in Java).
+
+- All judging machines are physically identical
+  to the laptops provided to you during the contest
+  (see [Systems](/systems)).
+
+- The memory limit is the same for all problems and is 2048 MB.
+
+- For problems with large I/O,
+  it is in your best interest to use buffered input and output in Java.
+  We have tried to make sure that the I/O-intensive problems
+  are solvable using even slow I/O.
+  But for some problems it can be more difficult to get accepted
+  using e.g. the `Scanner` class,
+  since it consumes a significant amount of valuable CPU time
+  that could otherwise be used by your algorithm.
+  In most cases, a `BufferedReader` will be more efficient.
+  Also consider explicit output buffering
+  (e.g., by using the `BufferedWriter` class)
+  to avoid flushing every line of output separately.
+
+- We guarantee that every problem is solvable in C++, Java, and Python 3.
+  There is no such guarantee for Kotlin.
```

Comment:
if kotlin is not allowed for teams, clarify this

Reply:
Kotlin is allowed, as shown on 2022.nwerc.eu/systems, we just don't guarantee it will be fast enough for every problem :stuck_out_tongue: